src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCtscOrgRoleMappingTest.java

```diff
@@ -71,5 +71,56 @@ void shouldReturnIACCtscMappings(String roleId, String serviceCode, String expec
                 });
     }
 
+    @ParameterizedTest
+    @CsvSource({
+            "10,BFA1,'task-supervisor',Y,N",
+            "10,BFA1,'case-allocator',N,Y",
+            "10,BFA1,'task-supervisor,case-allocator',Y,Y",
+            "9,BFA1,'ctsc-team-leader',N,N"
```

Comment:
Can we expand to cover the other `roleId = 9` scenarios that are being added.

Reply:
addressed